src/orders/README.md

```diff
@@ -206,6 +206,39 @@ This creates two secrets managed by kubernetes, named `atp-db-creds` and `atp-jk
 The kubernetes deployment manifest `atp-spring-jpa.yml` references these secrets and provides them to the runtime environment of the containers. The application now references the properties through the environment variables instead of reading them from a file. Alternatively the config files themselves could have been provided and mounted as volumes in to the container.
 
 
+```bash
+kubectl create secret generic monitoring-secret \
+--from-literal=compartment_id="<your compartment_id>" \
+--from-literal=tenant_id="<your tenancy_id>" \
+--from-literal=user_id="<your user_id>" \
+--from-literal=fingerprint="<your fingerprint>" \
+--from-literal=passphrase="<your passphrase>" \
+--from-literal=monitoring_endpoint="<your monitoring endpoint>" \
+--from-literal=region="<your region>" \
+--from-file=oci_api_key=<your oci_api_key.pem>
+```
+
+The above secret is required to send out custom metric (payment failure) to OCI Monitoring using OCI Java SDK.
+
+### For Developer setup:
+
+# Build
+
+## Java: building application jar
+
+OCI-java-sdk is a dependency and needs to be manually uploaded to 
+the local maven repository. 
+* download oci-java-sdk 
+`wget https://github.com/oracle/oci-java-sdk/releases/download/v1.25.1/oci-java-sdk-1.25.1.zip`
```

Comment:
![image](https://user-images.githubusercontent.com/551007/97805391-6a6aa200-1c1b-11eb-82ab-16805bab6ca4.png)

We spend a lot of time automating everything to not create any manual step or manual dependency. This should be corrected and if really needed, need to be automated. NEVER be a "local maven repository" or a manual step. This also breaks our current CI.

Reply:
This is just for developer setup running on your laptop. Everything is automated using Dockerfile for production.

Reply:
The developer machine, CI and server should not be different. The build process should be the same, without manual steps.